aca_preview/preview.py

```diff
@@ -80,17 +106,24 @@ def preview_load(load_name, outdir=None, report_level='none', loud=False):
     default is "none", meaning no reports are generated.  A final option is
     "all" which generates a report for every obsid.
 
-    :param load_name: Name of loads
-    :param outdir: Output directory
+    :param load_name: name of loads
+    :param outdir: output directory
     :param report_level: report level threshold for generating acq and guide report
-    :param loud: Print status information during checking
+    :param roll_level: level threshold for suggesting alternate rolls
+    :param loud: print status information during checking
+    :param acas: list of ACATable objects (optional, instead of ``load_name``)
```

Comment:
I'm a little confused about the 'instead of' here, as load_name doesn't look really optional.

Reply:
Fixed by 6cf74a2